### PR TITLE
[v1.4.0 cherrypick] ONNX Interpolate Add Scales Params - 2

### PR DESCRIPTION
**Pull request by lara-hdr**
The original PR is #28324

We hope we can cover torchvision models in PyTorch ONNX exporter with release 1.4. This PR is part of it.

cc: @houseroad

### CODE REVIEW

**Comment by kostmo** [comment]
## CircleCI build failures summary
As of commit 01ae0ca:
* **1/1** failures introduced in this PR



## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=01ae0ca3c460bd031d86349662f3b970fc2246ea).

### 1 new failure recognized by patterns
The following build failures don't appear to be due to upstream breakage:
#### [See CircleCI build](https://circleci.com/gh/pytorch/pytorch/3953088) pytorch_linux_xenial_py3_6_gcc5_4_build (1/1)
**Step:** "Build" ([details](https://dr.pytorch.org/build-details.html?build_id=74240286))
<details>
<summary>
<code>Automatic merge failed; fix conflicts and then commit the result.</code>
</summary>

```
CONFLICT (add/add): Merge conflict in .circleci/docker/common/install_cache.sh 
Auto-merging .circleci/docker/common/install_cache.sh 
CONFLICT (add/add): Merge conflict in .circleci/docker/common/install_android.sh 
Auto-merging .circleci/docker/common/install_android.sh 
CONFLICT (add/add): Merge conflict in .circleci/docker/build.sh 
Auto-merging .circleci/docker/build.sh 
CONFLICT (add/add): Merge conflict in .circleci/config.yml 
Auto-merging .circleci/config.yml 
CONFLICT (add/add): Merge conflict in .circleci/cimodel/data/binary_build_definitions.py 
Auto-merging .circleci/cimodel/data/binary_build_definitions.py 
Automatic merge failed; fix conflicts and then commit the result. 
```

</details>



---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

**Comment by BowenBao** [comment]
@spandantiwari @houseroad @gchanan @lara-hdr 
I'm taking over the PR since Lara is on vacation. 
Submitted the change for naming changes, from scales_1 to scales_h, etc.

**Comment by lara-hdr** [comment]
Thank you @BowenBao!
@gchanan, the PR is updated with the suggested change.

**Comment by gchanan** [comment]
@BowenBao can you submit a patch against master with only the diff changes?  That will be easier to review, and then we can just cherry-pick both PRs on top of each other from 1.4.

File: aten/src/ATen/native/UpSample.h [hunk 1]
```diff
@@ -3,6 +3,43 @@
...
 
+
+/**
+ * Note [compute_scales_value]
+ * Note [area_pixel_compute_scale]
+ * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ * Interpolate with scale_factors can have different behaviors
+ * depending on the value of use_scale_factor:
+ *
+ * - With use_scale_factor = False (current default behavior):
```
Comment (gchanan) [inline aten/src/ATen/native/UpSample.h]: this comment looks out of date.

Comment (BowenBao) [reply aten/src/ATen/native/UpSample.h]: Do you mean `False` should be `None`? I don't have enough context here, but by looking at the code I think other than that the description is accurate. I would include this change if needed in the separate PR as well.

**Comment by BowenBao** [comment]
> @BowenBao can you submit a patch against master with only the diff changes? That will be easier to review, and then we can just cherry-pick both PRs on top of each other from 1.4.

Submitted the separate PR against master https://github.com/pytorch/pytorch/pull/31526. And reverted these changes for this PR.

**Pull request closed without merging**